Cargo.toml

```diff
@@ -21,15 +21,15 @@ rustdoc-args = ["--cfg", "docsrs"]
 default = []
 derive = ["dep:tauri-specta-macros"]
 javascript = ["dep:specta-jsdoc"]
-typescript = ["dep:specta-typescript"]
```

Comment:
Can you explain the changes in this file?

Reply:
the js.rs file has this use `use specta_typescript::{js_doc, ExportError, Typescript};`
So compiling without the specta_typescript dependency fails if only the javascript feature is set.
This is why the dependency is now not optional anymore and the 'dep:..' is not needed.